schedule/yast/raid/raid1_sle_gpt_uefi_aarch64.yaml

```diff
@@ -0,0 +1,101 @@
+name:           RAID1_gpt_uefi
+description:    >
+  Configure RAID 1 on the disks with GPT partition tables and UEFI using Expert Partitioner.
+  Creates EFI boot, root and swap partitions on each of the 4 disks and then uses
+  them for RAID 1.
+vars:
+  RAIDLEVEL: 1
+  YUI_REST_API: 1
+schedule:
+  suggested_partitioning:
+    - installation/partitioning/raid_gpt
+  software:
+    - installation/installation_settings/validate_default_target
+  system_preparation:
+    - console/hostname
+    - console/system_prepare
+    - console/force_scheduled_tasks
+    - shutdown/grub_set_bootargs
+    - console/validate_md_raid
+    - console/validate_raid
+test_data:
```

Comment:
Please, let's use test_data and include like in the previous file instead of copy/paste the test data section, we don't need to duplicate this information, so in follow-up task we just need to worry about the test schedule.
Apply this to all the test suite where you copy/paste test data, will make easy the review as well.